queue/03_09_get_price_not_fall_periods_queue.py

```
@@ -0,0 +1,25 @@
+from collections import deque
+prices = [1, 2, 3, 2, 3]
+
+queue = deque(prices)
+
+def get_price_not_fall_periods(prices):
```

Ruff (0.13.1)

Unused function argument: `prices`

(ARG001)